util/sample_collections.py

```diff
@@ -355,11 +355,14 @@ def samples_from_files(filenames, buffering=BUFFER_SIZE, labeled=None):
         If False: Ignores transcripts (if available) and always reads (unlabeled) util.audio.Sample instances.
         If None: Reads util.sample_collections.LabeledSample instances from sources with transcripts and
         util.audio.Sample instances from sources with no transcripts.
+    reverse : bool
+        If True, perform Interleaved ordering in the reverse duration order
     """
     filenames = list(filenames)
     if len(filenames) == 0:
         raise ValueError('No files')
     if len(filenames) == 1:
         return samples_from_file(filenames[0], buffering=buffering, labeled=labeled)
```

Comment:
Reversing has also to be done on the interleaved sets - so here `samples_from_file` and (as called from there) `SDB` and `CSV` classes have to do the actual reversing.
```
samples_from_file(filenames[0], buffering=buffering, labeled=labeled, reverse=reverse)
```